content/en/docs/Installation/installation-guide/prerequisites.md

```diff
@@ -40,9 +40,10 @@ supported Istio versions.
 
 |<div style="width:50px">Istio</div>|<div style="width:125px">Kiali</div>|Notes|
 |-------|------------------|---|
+|1.15   |1.55.1 or later   |   |
 |1.14   |1.50.0 or later   |   |
```

Comment:
I believe this is going to cause the compat matrix warning. This is what someone already complained about.

https://github.com/kiali/kiali/blob/master/config/version-compatibility-matrix.yaml#L4-L9

```
  versionRange:
    - meshVersion: "1.15"
      kialiMinimumVersion: "1.55.0"
      kialiMaximumVersion: ""
    - meshVersion: "1.14"
      kialiMinimumVersion: "1.50.0"
      kialiMaximumVersion: "1.54"
```

So 1.14 isn't `1.50.0 or later` it is `1.50.0 to 1.54`